### ISSUE TITLE
Feature: Add @bind-Start and @bind-End two-way data binding

### ISSUE THREAD

**Issue by ryanelian** [jdtcn/BlazorDateRangePicker#5]
Enables acquiring date without writing an event handler method for `OnRangeSelect`

https://docs.microsoft.com/en-us/aspnet/core/blazor/components?view=aspnetcore-3.0#chained-bind

----

Relevant example for this project:

```cs
<input @oninput="OnValueChanged" value="@Value" />

<p>
    Child: @FirstWord + @SecondWord
</p>


@code {
    string Value { get; set; }

    [Parameter]
    public string FirstWord { set; get; }

    [Parameter]
    public EventCallback<string> FirstWordChanged { set; get; }

    [Parameter]
    public string SecondWord { set; get; }

    [Parameter]
    public EventCallback<string> SecondWordChanged { set; get; }

    private async Task OnValueChanged(ChangeEventArgs e)
    {
        Value = e.Value?.ToString();
        if (Value == null)
        {
            return;
        }

        var words = Value.Split(" ", StringSplitOptions.RemoveEmptyEntries);
        FirstWord = words.FirstOrDefault();
        await FirstWordChanged.InvokeAsync(FirstWord);
        SecondWord = words.Skip(1).FirstOrDefault();
        await SecondWordChanged.InvokeAsync(SecondWord);
    }
}
```

Example simplified and designed for this project.

For now, let's pretend that `Value` is acquired when user selected the value on the DatePicker. This is invisible outside the component (internal stuff).

Now, pretend `Start` and `End` parameters are `FirstWord` and `SecondWord`. By registering and calling the event callback in the component, user can directly bind into the values instead of relying on `OnRangeSelect` event:

```html
<TestInput @bind-FirstWord="Word1" @bind-SecondWord="Word2"></TestInput>
<p>
    Parent: @Word1 + @Word2
</p>

@code {
    string Word1 { set; get; }

    string Word2 { set; get; }
}
```

----

When above example is implemented into the project, the expected result should be something like this:

```html
<DateRangePicker @bind-Start="MyVariable1" @bind-End="MyVariable2"></TestInput>
@code {
    DateTimeOffset MyVariable1 { set; get; }

    DateTimeOffset MyVariable2 { set; get; }
}
```

**Comment by jdtcn** [comment]
I've implemented two-way binding feature, and published it with 1.1.0 package version. 

Readme updated in accordance with these changes.

Thank you for your feedback, I really appreciate it.

**Comment by ryanelian** [comment]
Great, thanks. You rock.
